atlas/AtlasMl/atlasml/scripts/populate_embedings.py

```
@@ -0,0 +1,105 @@
+populate_weaviate.py
+--------------------
+
+Usage:
+    python populate_weaviate.py \
+        --exercises ./samples/exercises.csv \
+        --competencies ./samples/competencies.csv \
+        --relations ./samples/relations.csv
+"""
+
+from __future__ import annotations
+import argparse
```

Ruff (0.12.2)

`argparse` imported but unused

Remove unused import: `argparse`

(F401)